bin/publish

```diff
@@ -2,3 +2,4 @@
 set -e
 cd "$(dirname "$0")"/..
 aws s3 sync --delete "$(pwd)/_site/" "s3://docs.armory.io/"
+aws cloudfront create-invalidation --distribution-id E32DKCY2L8NJH --paths /*
```

Comment:
I think the /* needs to be quoted, otherwise bash will expand it before it gets to the aws command.